packages/application-shell/src/components/navbar/navbar-skeleton.tsx

```diff
@@ -0,0 +1,184 @@
+import type { ReactNode } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { designTokens } from '@commercetools-uikit/design-system';
+import { DIMENSIONS } from '../../constants';
+
+const NavBarLayout = (
+  props: TNavBarSkeletonProps & { children: ReactNode }
+) => {
+  return (
+    <nav
+      aria-busy={true}
+      css={css`
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        width: ${props.isExpanded ? '256px' : '80px'};
+      `}
+    >
+      {props.children}
+    </nav>
+  );
+};
+
+const NavBarHeader = styled.div`
```

Comment:
**nit**: I would move all the styled components to a `navbar-skeleton.styles.tsx` file so we separate styling from structure and keep files smalls but it's up to you.